ListApp/Sources/ListApp.Contracts/Models/ListItem.cs

```diff
@@ -6,5 +6,7 @@ public class ListItem
     {
         public Guid Id { get; set; }
         public string Text { get; set; }
+        public DateTime? Created { get; set; }
```

Comment:
Since both Created and LastEdited propperties are not going to be consumed by JS frontend, newly posted and edited items coming from the frontend have no way of including this data. Having them send null instead of default value for the DateTime type seemed more reasonable to me.

Reply:
Right, you're using that model also as a view-model for insertions and updates. Ok, in that case, a null/undefined is better than some arbitrary value to be discarded.